src/log_parser.py

```diff
@@ -375,9 +377,9 @@ def _filehandle(self):
         if not self._fh or self._is_closed():
             filename = self._rotated_logfile or self.filename
             if filename.endswith('.gz'):
-                self._fh = gzip.open(filename, 'r', encoding="utf-8")
+                self._fh = gzip.open(filename, 'rb')
             else:
-                self._fh = open(filename, "r", 1, encoding="utf-8")
+                self._fh = open(filename, "rb")
```

Comment:
Can you explain this change? It probably doesn't need utf-8 anymore (I used to display steam names instead of hero names) but I'm just curious if this was breaking something